test/llvm_ir_correct/fp_reduce_intrinsic.f90

```diff
@@ -2,28 +2,31 @@
 ! See https://llvm.org/LICENSE.txt for license information.
 ! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-! RUN: %flang -O3                             -emit-flang-llvm %s -o - | FileCheck %s  --check-prefixes=CHECK
-! RUN: %flang -Ofast                          -emit-flang-llvm %s -o - | FileCheck %s  --check-prefixes=FAST
-! RUN: %flang -O3 -Mx,216,0x8                 -emit-flang-llvm %s -o - | FileCheck %s  --check-prefixes=NSZ
-! RUN: %flang -O3 -Mx,216,0x10                -emit-flang-llvm %s -o - | FileCheck %s  --check-prefixes=REASSOC
-! RUN: %flang -O3 -Mx,216,0x8 -Mx,216,0x10    -emit-flang-llvm %s -o - | FileCheck %s  --check-prefixes=NSZ_REASSOC
-! RUN: %flang -Ofast -Mx,216,0x8 -Mx,216,0x10 -emit-flang-llvm %s -o - | FileCheck %s  --check-prefixes=FAST
+! REQUIRES: non-ms-sdk
+! REQUIRES: llvm-13
+
+! RUN: %flang -O3                             -emit-flang-llvm %s -o /dev/stdout | FileCheck %s  --check-prefixes=CHECK
+! RUN: %flang -Ofast                          -emit-flang-llvm %s -o /dev/stdout | FileCheck %s  --check-prefixes=FAST
+! RUN: %flang -O3 -Mx,216,0x8                 -emit-flang-llvm %s -o /dev/stdout | FileCheck %s  --check-prefixes=NSZ
+! RUN: %flang -O3 -Mx,216,0x10                -emit-flang-llvm %s -o /dev/stdout | FileCheck %s  --check-prefixes=REASSOC
+! RUN: %flang -O3 -Mx,216,0x8 -Mx,216,0x10    -emit-flang-llvm %s -o /dev/stdout | FileCheck %s  --check-prefixes=NSZ_REASSOC
+! RUN: %flang -Ofast -Mx,216,0x8 -Mx,216,0x10 -emit-flang-llvm %s -o /dev/stdout | FileCheck %s  --check-prefixes=FAST
 
 real function acc(arr1,arr2)
   real :: arr1(:)
   real :: arr2(:)
   acc = SUM(arr1 * arr2)
-! CHECK-NOT: call fast float @llvm.fmuladd.f32
-! CHECK-NOT: call nsz float @llvm.fmuladd.f32
-! CHECK-NOT: call nsz reassoc float @llvm.fmuladd.f32
-! FAST-NOT: call nsz float @llvm.fmuladd.f32
-! FAST-NOT: call reassoc float @llvm.fmuladd.f32
-! NSZ: call nsz float @llvm.fmuladd.f32
-! NSZ-NOT: call fast float @llvm.fmuladd.f32
-! NSZ-NOT: call reassoc {{.*}}float @llvm.fmuladd.f32
-! REASSOC-NOT: call fast float @llvm.fmuladd.f32
-! REASSOC-NOT: call nsz {{.*}}float @llvm.fmuladd.f32
-! REASSOC: call reassoc float @llvm.fmuladd.f32
-! NSZ_REASSOC: call nsz reassoc float @llvm.fmuladd.f32
-! NSZ_REASSOC-NOT: call fast float @llvm.fmuladd.f32
+! CHECK-NOT: {{[[:space:]]}}fast
```

Comment:
What problem did this fix? Can you describe the situation that would cause this test to fail incorrectly?

You might want to make sure that the word "fast" terminates too, e.g.
```
! CHECK-NOT: {{[[:space:]]fast[[:space:]]}}
```

Reply:
those command lines have "fp-contract"="fast", it's not the 'fast' we don't want to see here. Looking for a space before 'fast' is minimalistic yet sufficient.

Reply:
Actually my question was, what was the problem with `CHECK-NOT: call fast float @llvm.fmuladd.f32`?

Reply:
now I can see that this test file is entirely redundant, it tests the same things as fp_reduce.f90, while it was supposed to test attributes of the calls to the LLVM intrinsics, these are not emitted by the upstream compiler.